minecraft/protocol/events.go

```diff
@@ -110,14 +110,28 @@ func (c *CauldronUsedEventData) Marshal(r IO) {
 type PlayerDiedEventData struct {
 	// AttackerEntityID ...
 	AttackerEntityID int32
+	// AttackerVariant ...
+	AttackerVariant int32
 	// EntityDamageCause ...
 	EntityDamageCause int32
+	// InRaid ...
+	InRaid bool
 }
 
 // Marshal ...
-func (p *PlayerDiedEventData) Marshal(r IO) {
+func (p *PlayerDiedEventData) Marshal(w *Writer) {
```

Comment:
I don't think this compiles. Implementations of the `protocol.EventData` interface must (only) implement a `Marshal(r IO)` interface.

Reply:
Oh, I didn't notice that